### DISCUSSION TITLE
Container Env: installing plugin on the base image : ghcr.io/aquasecurity/trivy:0.56.1

### DISCUSSION THREAD

**Discussion by kabilanmani93** [fatihtokus/scan2html#106] [Q&A] [answered]
**issue**:
/ # /root/.trivy/plugins/scan2html/scan2html.sh
/bin/sh: /root/.trivy/plugins/scan2html/scan2html.sh: not found

**checked the file:**
/ # ls -la /root/.trivy/plugins/scan2html/
total 932
drwxr-xr-x    2 root     root          4096 Oct  5 05:11 .
drwxr-xr-x    3 root     root          4096 Oct  5 05:11 ..
-rw-r--r--    1 root     root         11357 Sep 21 21:11 LICENSE
-rw-r--r--    1 root     root           877 Oct  5 05:11 plugin.yaml
-rw-r--r--    1 root     root        921020 Sep 21 21:12 report_template.html
-rwxr-xr-x    1 root     root          7049 Sep 21 21:11 scan2html.sh

permission and file exist, but unable to run .

**Comment by fatihtokus** [comment]
Thanks for the issue, we will check and revert back to you

**Reply by princechennai** [reply]
Is there any update on this? I'm having the same issue

**Comment by fatihtokus** [comment, marked as answer]
Hi @kabilanmani93 , @princechennai ;

Thanks for your patience. At last, we got to the bottom of it. Basically, scan2html requires bash which doesn't come with the base image of trivy. So we need to install it as shown below. Please try and let me know

docker run --rm \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -v $(pwd):/output \
  --entrypoint /bin/sh \
  aquasec/trivy:0.56.2 -c "_apk update && apk add bash && bash_ -c 'trivy plugin run scan2html image nginx:latest /output/report.html'"

**Reply by princechennai** [reply]
That worked. Thanks for your time on this

**Reply by fatihtokus** [reply]
No problem @princechennai . Please consider giving us a GitHub star ⭐️. Thank you!

Regards,
Fatih
